src/pages/BuyCrypto/index.tsx

```diff
@@ -260,33 +262,22 @@ function BuyCrypto() {
           </Flex>
 
           <Flex sx={{ gap: '20px' }} marginTop="24px" flexDirection="column" justifyContent="center">
-            <DownloadWalletRow href="https://metamask.io/download/" target="_blank" rel="noopener noreferrer">
-              <Image src={metamask} width="24px" />
-              MetaMask
-            </DownloadWalletRow>
-            <DownloadWalletRow href="https://wallet.coin98.com/" target="_blank" rel="noopener noreferrer">
-              <Image src={c98} width="24px" />
-              Coin98
-            </DownloadWalletRow>
-
-            <DownloadWalletRow href="https://walletconnect.com/" target="_blank" rel="noopener noreferrer">
-              <Image src={walletConnect} width="24px" />
-              WalletConnect
-            </DownloadWalletRow>
-
-            <DownloadWalletRow href="https://www.coinbase.com/wallet" target="_blank" rel="noopener noreferrer">
-              <Coinbase width={'24px'} height="24px" />
-              <Text>Coinbase Wallet</Text>
-            </DownloadWalletRow>
-
-            <DownloadWalletRow
-              href="https://www.ledger.com/ledger-live/download"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              <Ledger width={'24px'} height="24px" />
-              <Text>Ledger</Text>
-            </DownloadWalletRow>
+            {Object.values(SUPPORTED_WALLETS)
+              .filter(e => e.installLink)
+              .map(item => (
+                <DownloadWalletRow
+                  key={item.installLink}
+                  href={item.installLink}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <Image
+                    width="24px"
+                    src={require(`../../assets/images/${isDarkMode ? '' : 'light-'}${item.iconName}`).default}
```

Comment:
looks horrible IMO. Should import both image then conditional use those image here